### ISSUE TITLE
Installation of all color versions not working

### ISSUE THREAD

**Issue by ghost** [yeyushengfan258/BigSur-icon-theme#10]
When I run "install.sh -a" there is only the "BigSur-black" themes created.
![bigsur-icon-theme](https://user-images.githubusercontent.com/76121935/137587139-2fa1f6a5-1015-423b-9ba4-ee77d4354090.png)

**Comment by yeyushengfan258** [comment]
Yes, there is no problem with this, there are only 2 colors for the time being.
